src/components/Chart/TimeBars.docs.md

```diff
@@ -991,15 +991,29 @@ age,date,value
       "value": 6000,
       "label": "Schätzung",
       "showValue": false,
-      "position": "bottom"
+      "position": "bottom",
+      textAlignment: 'right',
```

Comment:
i know this is super annoying with markdown files, but some things having double-quotes and some not 😱